scalameta/scalameta/shared/src/test/scala/scala/meta/tests/quasiquotes/ErrorSuite.scala

```diff
@@ -4,14 +4,15 @@
 import org.scalatest._
 import org.scalameta.tests._
 import typecheckError.Options.WithPositions
+import compat.Platform.EOL
 
 class ErrorSuite extends FunSuite {
   test("val q\"type $name[$A] = $B\"") {
     assert(typecheckError("""
       import scala.meta._
       import scala.meta.dialects.Scala211
       val q"type $name[$X] = $Y" = q"type List[+A] = List[A]"
-    """) === """
+    """).replace("\r", "") === """
```

Comment:
@olafurpg maybe you can shed some light on what's happening here: typecheckError seems to emit mixed line endings. My best guess is that line endings are getting inlined from a different compile-time Platform in XtensionPositionFormatMessage but by best guess isn't very good.

Reply:
Hmm, I am not sure what is happening here. If typecheckError is defined in scalameta, then I'm happy to move the replace there instead of at every callsite.